docs/en/04_Changelogs/3.7.0.md

```diff
@@ -0,0 +1,26 @@
+# 3.7.0
+
+### Versioned cache segmentation
+
+The cache API now maintains separate cache pools for each versioned stage. This prevents users from caching draft data and then exposing it on the live stage.
```

Comment:
Call it `SS_Cache` API, to be clear about which cache you mean (e.g. not HTTP cache).

Also, I think the intent for this change should be more obvious. Suggested rewording:

> This prevents developers from caching draft data and then accidentally exposing it on the live stage without potentially required authorisation checks. Unless you rely on caching across stages, you don't need to change your own code for this change to take effect. Note that cache keys will be internally rewritten, causing any existing cache items to become invalid when this change is deployed.